features/mbedtls/targets/TARGET_NUVOTON/TARGET_M480/sha/sha1_alt.h

```diff
@@ -37,8 +31,7 @@ struct mbedtls_sha1_context_s;
 /**
  * \brief          SHA-1 context structure
  */
-typedef struct mbedtls_sha1_context_s
-{
+typedef struct mbedtls_sha1_context_s {
     int ishw;
     crypto_sha_context hw_ctx;
     mbedtls_sha1_sw_context sw_ctx;
```

Comment:
Is it possible to have both hardware and software contexts active at the same time? If this is not the case, perhaps its better to use a union?

Reply:
@andresag01 H/W and S/W contexts are needed at the same time in e.g. `mbedtls_sha1_clone`.